certified-connectors/Office365GroupsMail/apiProperties.json

```diff
@@ -0,0 +1,50 @@
+{
+  "properties": {
+    "capabilities": [
+      "actions",
+      "triggers"
+    ],
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "aad",
+          "clientId": "<<Client ID placeholder>>",
+          "scopes": [
+            "Group.ReadWrite.All offline_access"
```

Comment:
Isn't Group.Read.All enough?

Reply:
Maybe for this operation? DeleteConversationThread

Reply:
Yes, I'll remove the `offline_access` scope in the next iteration.

Reply:
So ReadWrite is needed for DeleteConversationThread?

Reply:
The operation `DeleteConversationThread` needs `Group.ReadWrite.All`; [here](https://docs.microsoft.com/en-us/graph/api/group-delete-thread?view=graph-rest-1.0&tabs=http#permissions) is the documentation.

Apologies, I did not refresh the page and I skipped your last comment.